[Doc] Add multiple bot installation (#1732)
Drakkar-Software/OctoBot: docs/source/pages/With-Docker.rst

You can now open the OctoBot web interface at https://YOUR_IP_ADDRESS.

Start multiple OctoBots with docker
---------------------------------------

To run a second OctoBot on the same computer :

1. Create a new directory and enter it

2. Start OctoBot's web interface on a new port by changing "-p" option

.. code-block:: bash

   docker run -itd --name OctoBot -p 8000:5001 -v $(pwd)/user:/octobot/user -v $(pwd)/tentacles:/octobot/tentacles -v $(pwd)/logs:/octobot/logs drakkarsoftware/octobot:stable

In this example, the second OctoBot's web interface is accessible at http://127.0.0.1:8000.

> Any port can be used by those already used by an OctoBot or by the system.

Reviewer: ```suggestion
Any port can be used except those already used by another OctoBot or any software on your system.
```
